components/TutorialsList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <TutorialsSubList
+    v-for="tutorialList in tutorials"
+    :key="tutorialList.title"
+    :tutorials="tutorialList.tutorialLinks"
+    :title="tutorialList.title"
+  />
+</template>
+<script setup lang="ts">
+import type { TutorialList } from '@/types'
+const { t } = useI18n()
+
+// TODO: add urls
+const tutorials: Ref<Array<TutorialList>> = ref([
```

Comment:
Does this need to be reactive?

```suggestion
const tutorials: <Array<TutorialList>> = [
```